### PR TITLE
Fix some issues

### PR DESCRIPTION
**Pull request by azizullahsaeidi**
On this PR I have fixed some issues and add Recaptcha v3.

1. Remove facebook sdk
2. Fix GlossaryListTest's namespace 
3. Create .env.example
4. Add subscriber to registration form and add the text on language files
5. Add reCaptcha V3
6. Create RecaptchaRule for validation base on [documentation](https://developers.google.com/recaptcha/docs/verify) 
7. Fix ddlibrary failed issues

Note: please create reCaptcha site and secret keys from this [link](https://www.google.com/recaptcha/admin/create)

> Please run `php artisan migrate` , ` composer install` and `npm run production`

### CODE REVIEW

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -216,6 +216,14 @@
                 @endif
             </ul>
         </nav>
+
+        <div>
+            <form action="{{ route('subscribe') }}" method="POST">
```
Comment (uyscuti-wiki) [inline resources/views/home.blade.php:221]: We want this form to be protected by reCAPTCHA as well.

File: resources/views/home.blade.php [hunk 2]
```diff
@@ -216,6 +216,14 @@
...
+                @csrf
+                <input type="email" name="email" placeholder="@lang('Enter Your Email')" class="form-control display-inline-block">
+                <button type="submit" class="form-control submit-button btn btn-primary  display-inline-block p-7"> @lang("Subscribe") </button>
+            </form>
```
Comment (uyscuti-wiki) [inline resources/views/home.blade.php:225]: If I am being honest, I am not a fan of placing a text field here. Can we remove the text field and change the 'submit' button to a link like the rest of the links? You can redirect users to a different page where we ask their name and email (if not logged in). If they are logged in, auto populate the fields.

**Review by uyscuti-wiki — CHANGES_REQUESTED** [review]
Thank you Aziz. This looks pretty good to me. Please address the inline comments. Additionally I have a few comments as well.

1. The login, and the contact us forms must be reCAPTCHA protected. 
2. The login page fields appear wonky, which needs to be fixed. 
3. On the register page, move the 'sign in' link from the form to below the header 'Register an account' in smaller text. The text could be - 'Or sign in if you already have an account'. 
image

Thank you!

File: database/migrations/2023_12_07_125201_create_subscribers_table.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('subscribers', function (Blueprint $table) {
+            $table->id();
+            $table->string('email')->unique();
```
[the remark this answers is not in the document]
Comment (uyscuti-wiki) [reply database/migrations/2023_12_07_125201_create_subscribers_table.php:18]: Might be worth adding the `user_id` (if available) and `name` as columns.

File: config/services.php [hunk 4]
```diff
@@ -53,4 +53,10 @@
         'redirect' => 'https://library.darakhtdanesh.org/login/google/callback/',
     ],
 
+    // Configure Google ReCAPTCHA v3 key+secret
+    'recaptcha_v3' => [
+        'site_key' => env('RECAPTCHAV3_SITEKEY'),
+        'secret_key' => env('RECAPTCHAV3_SECRET'),
+    ]
+
```
[the remark this answers is not in the document]
Comment (azizullahsaeidi) [reply config/services.php:61]: Yes we should remove it. I will do it. Thanks

File: lang/en.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+    "Receive our occasional newsletters and announcements (your email will be shared with MailChimp)":"Receive our occasional newsletters and announcements (your email will be shared with <a href='<a href='https://mailchimp.com/legal/'>MailChimp</a>)",
+    "Your email will be shared with MailChimp. their privacy policy": "Your email will be shared with MailChimp. <br> <a href='https://mailchimp.com/legal/'> Their Privacy Policy",
+    "You'll receive our monthly newsletter with our latest updates, highlights, top resources, educational content and other relevant news.": "You'll receive our monthly newsletter with<br> our latest updates, highlights, top resources,<br> educational content and other relevant news.",
+    "If you registered using a phone number, please contact us.": "If you registered using a phone number, please <a href='/contact-us'>contact us</a>."
+}
```
Comment (uyscuti-wiki) [inline lang/en.json:6]: Please add a newline at EOF here.

**Review by uyscuti-wiki — APPROVED** [review]
Looks great! Thank you Aziz.

**Pull request merged**